lib/lotus/model/migrator.rb

```diff
@@ -292,6 +292,8 @@ def self.connection
         Sequel.connect(
           configuration.adapter.uri
         )
+      rescue Sequel::AdapterNotFound
+        raise MigrationError.new("Current adapter (#{ configuration.adapter.type }) doesn't support database sequel operations.")
```

Comment:
What do you think about rewording it to `does not support SQL database operations`?